jekyll/_cci2/[WIP] runner-overview.adoc

```diff
@@ -0,0 +1,96 @@
+= CircleCI Runner Overview
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Introduction
+
+CircleCI Runner enables you to use your own infrastructure for running jobs. This means you will be able to build and test on a wider variety of architectures and environments, in addition to providing you with additional control over the security of the environment.
+
+== Runner Use Cases
+
+There are two key use cases we are aiming to meet with the runner:
```

Comment:
* I think we generally stay away from using "we"? Maybe '... two key use cases for which runner is a good fit' or something?

* and maybe "CircleCI runner" rather than "the runner"? Not sure on that though...

Reply:
Fixed both issues